web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/AutoDeploy.tsx

```diff
@@ -197,6 +197,12 @@ export function AutoDeploy({ toggleDeployMethod }: DeployServiceProp) {
                     <Icons.Warning ml={1} color="error.main" />
                     Encountered Error: {attempt.statusText}
                   </TextIcon>
+                  <Text mt={2}>
+                    <b>Note:</b> If this is your first attempt, it might be that
+                    AWS has not finished propagating changes from{' '}
+                    <Mark>Step 1</Mark>. Try waiting a minute before attempting
+                    again.
```

Comment:
How do they "attempt again" here? Does the `Deploy Teleport Service` button need to be re-enabled? It's not clear, but it seems as though it may be disabled in your screenshot.

Reply:
by clicking on the "deploy teleport service" button

non error state: 
<img width="1017" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/aa0af5e6-e6e6-42a8-89d6-1bb244891c02">

error state:
![image](https://github.com/gravitational/teleport/assets/43280172/1d57de40-8fe2-4116-8e32-a7dd72e2a4ed)